src/inventory_ui.cpp

```diff
@@ -449,9 +752,38 @@ void inventory_selector::display(bool show_worn) const
     } else {
         print_inv_weight_vol(g->u.weight_carried(), g->u.volume_carried(), g->u.volume_capacity());
     }
-    if (!multidrop && !compare) {
-        mvwprintw(w_inv, 1, 61, _("Hotkeys:  %d/%d "),
-                  g->u.allocated_invlets().size(), inv_chars.size());
+    
+    if(OPTIONS["INV_DISPLAY"] == "list") {
+        const size_t &current_page_offset = in_inventory ? current_page_offset_i : current_page_offset_w;
+        if (multidrop) {
+            mvwprintw(w_inv, 1, 0, _("To drop x items, type a number and then the item hotkey."));
+        }
+        print_left_column();
+        if(show_worn) {
+            print_middle_column();
+        }
+        print_right_column();
+        const size_t max_size = in_inventory ? items.size() : worn.size();
+        const size_t max_pages = (max_size + items_per_page - 1) / items_per_page;
+        mvwprintw(w_inv, items_per_page + 4, 1, _("Page %d/%d"), current_page_offset / items_per_page + 1,
+                  max_pages);
+        if (!multidrop && !compare) {
+            mvwprintw(w_inv, 1, 61, _("Hotkeys:  %d/%d "),
+                      g->u.allocated_invlets().size(), inv_chars.size());
+        }
+        
+    } else if(OPTIONS["INV_DISPLAY"] == "grid") {
+        if (multidrop) {
+            if(TERMX > 100)
```

Comment:
{} are mandatory even for one-line conditions. More general note, you have to follow official code style.